Tests/Reflection/TypeExtensionTests.cs

```diff
@@ -1,32 +1,11 @@
-using NUnit.Framework;
-using Anvil.CSharp.Reflection;
+
 
 namespace Anvil.CSharp.Tests
 {
     public static class TypeExtensionTests
     {
```

Comment:
Is this just an empty file now? Can it be removed?

Reply:
#131 Incoming!